### PR TITLE
feat(map/basic): do not show label to our current location

### PR DESCRIPTION
**Pull request by AitorRodriguez990**
## Description

Show label only on children and brothers. Until now, we were showing the label for the current location too.

## Preview

In this example, we removed "El Masnou" label and only show its children.

image

### CODE REVIEW

File: components/map/basic/src/leaflet/shapes/Polygons.js [hunk 1]
```diff
@@ -67,17 +74,26 @@ export default class SearchMapPolygons {
...
-          layer
-            .bindTooltip(layer.feature.properties.LocationName, {
-              permanent: true,
-              direction: 'center',
-              className: fitBound ? 'is-fit-bound' : ''
-            })
-            .openTooltip()
+          const {Code, LocationName} = layer.feature.properties
+
+          that.currentGeoCode !== Code &&
```
Comment (ruben-martin-lozano) [inline components/map/basic/src/leaflet/shapes/Polygons.js]: Pondría una prop (boolean), para decidir si mostrar o no el tooltip para el current code

**Comment by midudev** [comment]
We keep feeding the component with logic intended to be used only for FC. All of this should be placed outside this component, ideally on the FC component side.

File: components/map/basic/src/leaflet/shapes/Polygons.js (hunk 1 above)
Comment (AitorRodriguez990) [reply components/map/basic/src/leaflet/shapes/Polygons.js]: Done!

**Comment by AitorRodriguez990** [comment]
@midudev `properties` is not a Real Estate property, all about `layer`, `feature` or ` properties` are Leaflet properties.

**Pull request merged**